src/serac/physics/tests/serac_solid_functional.cpp

```diff
@@ -375,12 +372,12 @@ void functional_parameterized_solid_test(double expected_disp_norm)
   EXPECT_NEAR(expected_disp_norm, norm(solid_solver.displacement()), 1.0e-6);
 }
 
-TEST(SolidFunctional, 2DLinearStatic) { functional_solid_test_static<1, 2>(1.5110743593501033); }
-TEST(SolidFunctional, 2DQuadStatic) { functional_solid_test_static<2, 2>(2.1864070695817928); }
-TEST(SolidFunctional, 2DQuadParameterizedStatic) { functional_parameterized_solid_test<2, 2>(2.1864070695817928); }
+TEST(SolidFunctional, 2DLinearStatic) { functional_solid_test_static<1, 2>(1.5110529858788075); }
```

Comment:
Note - these norms changed because the integration occurs on the reference instead of the deformed configuration now. This is actually one of the side effects that inspired this PR.

Reply:
Yes, it was patch test failures that caught this. Once that branch is merged, most of these tests will be deleted.